packages/components/src/select-control/list.js

```diff
@@ -116,9 +116,11 @@ class List extends Component {
 				return;
 
 			case TAB:
-				this.select( options[ selectedIndex ] );
-				setExpanded( false );
-				onSearch( null );
+				if ( selectedIndex > -1 ) {
```

Comment:
Given the other changes, I don't think this check is needed.

Reply:
Actually we do, since pressing tab on the component will attempt to select anyways. Doesn't break the component but it does prompt an error on [this line](https://github.com/woocommerce/woocommerce-admin/blob/d167204498f57a74b45ae1a668050ed8674d6bb8/packages/components/src/select-control/list.js#L48) since the `option` is `undefined`.